godtools/App/Features/AppLanguage/Domain/Interface/GetDownloadableLanguagesListRepositoryInterface.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  GetDownloadableLanguagesListRepositoryInterface.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 12/15/23.
+//  Copyright © 2023 Cru. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+protocol GetDownloadableLanguagesListRepositoryInterface {
+    
+    func getDownloadableLanguagesPublisher(currentAppLanguage: AppLanguageDomainModel) -> AnyPublisher<[DownloadableLanguageListItemDomainModel], Never>
+    func observeLanguagesChangedPublisher() -> AnyPublisher<Void, Never>
```

Comment:
Left a comment else where, but after our last discussion on which architecture format to go with we won't need these exposed observers.  All data layer observing will now be pushed to the interface implementation.